data/hai/hai reveal 1 intro.txt

```diff
@@ -13,6 +13,11 @@
 
 ########################################################################
 
+disable mission
+	"Hai Secret Leaker Begins"
+	"Hai Rescue: Yeertle Family"
+	"Hai Rescue: Ooonem"
```

Comment:
```suggestion
```

Both of these missions are quite suitable to have in-game as purely stand alone missions. In my playthrough I encountered the Yeertle family well before I got embroiled in the larger HR story, and I loved it.

These missions make sense to be happening. Let's face it, even if the entirety of HR got scrapped, these rescue missions would still fit because, well, humans being humans, even if there isn't a big organization kidnapping people, there's still going to be conspiracy nuts and trophy hunters and rich people who will pay for the novel.

Look at how much effort humanity has spent tracking down Bigfoot and other mythical creatures. Even a single picture of a Hai that isn't obviously doctored, posted on some random disreputable forum, and there's going to be at least a small horde of people trying to catch them or at least get a picture of them, for various reasons.

I've got 84 no-name crew on my Dreadnought going back and forth here whenever I feel like visiting the Hai. I'd judge that the odds that none of them have ever posted a picture to social media or showed a friend a picture of them on one of the many beautiful places in hai space is virtually zero. And that's just my ship alone. Extend that out to the probably hundreds of thousands of people on all the various merchants moving around? It's a miracle (or a testament to the efficacity of Republic Intelligence) that the Hai aren't a regular feature of the tabloids.

Reply:
Agreed, these missions don't appear to rely on the overarching HR story to any major degree, and don't lead into any HR missions. They work perfectly well as standalone missions to complement what is already vanilla and would serve as a good tie-in for the larger Hai Reveal plotline for Steam users who experience these missions first. 
Unless there's a specific reason why these have to be disabled, I think we should keep them for this release.